css/main.css

```diff
@@ -1,3 +1,391 @@
+    top: 0;
+    left: 0;
+}
+
+
+.header ul {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    overflow: hidden;
+    background-color: black;
+    display: block;
+    text-align: center;
+    
+}
+
+.header li {
+    padding: 20px 30px;
+    border: 1px solid #333;
+    
+}
+.header li a {
+   text-decoration: none;
+   cursor: pointer;
+   color: white;
+   
+}
+.header li a:hover,
+.header .menu-btn:hover {
+    color: red;
+} 
+
+.header .menu {
+    transition: max-height .2s ease-out;
+    max-height: 0px;
+    position: absolute;
+    top: 0;
+    margin-top: 68px;
+    left: 0;
+    flex-direction: column;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    
+}
+.header .menu-icon {
+    cursor: pointer;
+    display: block;
+    float: right;
+    padding: 28px 30px;
+    position: relative;
+    cursor: pointer;
+}
+.header .menu-icon .navicon {
+    background: black;
+    display: block;
+    height: 2px;
+    position: relative;
+    transition:  .2s ease-out;
+    width: 18px;
+}
+.header .menu-icon .navicon:before,
+.header .menu-icon .navicon:after {
+    background: black;
+    content: '';
+    display: block;
+    height: 100%;
+    position: absolute;
+    transition: all .2s ease-out;
+    width: 100%;
+
+}
+.header .menu-icon .navicon:before {
+    top: 8px;
+}
+.header .menu-icon .navicon:after {
+    top: -8px;
+}
+.header .menu-btn {
+    display: none;
+
+}
+.header .menu-btn:checked ~ .menu  {
+    max-height: 240px;
+    transition: all 1s cubic-bezier(.5, 1, .22, 1);
+
+}
+.header .menu-btn:checked ~ .menu-icon .navicon {
+    background: transparent;
+    
+  }
+  .header .menu-btn:checked ~ .menu-icon .navicon:before {
+    transform: rotate(-45deg);
+  }
+  
+  .header .menu-btn:checked ~ .menu-icon .navicon:after {
+    transform: rotate(45deg);
+  }
+  .header .menu-btn:checked ~ .menu-icon:not(.steps) .navicon:before,
+.header .menu-btn:checked ~ .menu-icon:not(.steps) .navicon:after {
+  top: 0;
+}
+.contact-icons .mobile {
+    visibility: hidden;
+}
+.fa {
+    text-decoration: none;
+    color: red;
+    margin-left: 5px;
+}
+.st-title {
+    justify-content: space-around;
+}
+.centered {
+    margin-top: 30px;
+    font-size: 1em;
+}
+.centered p {
+   text-align: center;
+}
+.since {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 50px;
+}
+.products {
+    justify-content: space-around;
+    align-items: center;
+}
+.row img {
+    width: 100%;
+}
+.since img {
+    width: 50px;
+    padding: 10px;
+}
+
+h2 {
+    text-align: center;
+}
+.box h2,h3,p{
+    text-align: center;
+}
+
+.about{
+    background-color: rgba(0,0,0, .3);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.3em;
+    text-align: center;
+}
+.about h1{
+    color: red;
+}
+
+.grid-img {
+    position: relative;
+}
+.sale {
+    background-color: rgba(0,0,0, .3);
+    width: 100%;
+    position: absolute;
```

Comment:
Be careful with the use of position: absolute - it is causing your elements to overlap each other. 
Add images first and once the elements on the page have their own height, you'll be able to display text on top of them if needed.